spec/project/default_config_spec.rb

```diff
@@ -6,10 +6,11 @@
   let(:cop_names) do
     namespaces = {
       'rspec' => 'RSpec',
+      'capybara' => 'Capybara',
       'factory_girl' => 'FactoryGirl'
     }
     glob = SpecHelper::ROOT.join('lib', 'rubocop', 'cop',
-                                 'rspec', '{,factory_girl/}*.rb')
+                                 'rspec', '{capybara/,,factory_girl/}*.rb')
```

Comment:
Requiring these two changes might get messy quickly if you anticipate adding more namespaces, but I went with the simplest thing for now.